frontend/packages/kubevirt-plugin/src/components/vms/vm-resource.scss

```diff
@@ -1 +1,6 @@
 @import "./resource-summary-description";
+
+.kubevirt-vm-details__vm-details-item {
```

Comment:
If we are to change the position of the Alert to be on top of the title, theres no need to create a new scss file.
you can use a class like `.co-m-form-row`. 
check `_common.scss` for more classes.